src/System/PosixCompat/Process.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE CPP #-}
+
+{-|
+This module makes the operations exported by @System.Posix.Process@
+available on all platforms. On POSIX systems it re-exports operations from
```

Comment:
I suppose this is a more a statement of intent than of actual function.
Currently, it only re-exports a single operation from `System.Posix.Process`.
Maybe it  could be reformulated as `This module intends to make the operations of @System.Posix.Process@ available on all platforms.`  What do you think?

Reply:
That sounds good; I’m on favour of iteratively improving the docs. Before the next version we could consider rewording the other modules as well.